README.md

```diff
@@ -58,27 +59,22 @@ You can omit either of the two directories, and Meson will substitute
 the current directory and autodetect what you mean. This allows you to
 do things like this:
 
-`cd source_root; mkdir builddir; cd builddir; meson ..`
-
-or
-
-`cd source_root; mkdir builddir; meson builddir`
+```zsh
+cd <source root>
+meson builddir
+```
 
 To compile, cd into your build directory and type `ninja`. To run unit
 tests, type `ninja test`.
 
-Install is the same but it can take an extra argument:
-
-`DESTDIR=/destdir/path ninja install`
-
-`DESTDIR` can be omitted. If you are installing to system directories,
-you may need to run this command with sudo.
-
```

Comment:
I'm not sure that dropping the note about DESTDIR is a good idea, specifically because meson uses DESTDIR as an environment variable, not as an argument to make like you'd do with autotools. It could certainly be *clearer*, but I'm not in favor of dropping it.

Reply:
I understand that but there is already some documentation on the DESTDIR environment variable, so that is why I added a link to the [running Meson page](https://mesonbuild.com/Quick-guide.html#using-meson-as-a-distro-packager).